### DISCUSSION TITLE
How to use scrollbar style

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#6477] [Help]
I want to change the `scrollbar` style. I copied the `scrollbar` into the class and it didn’t take effect. I didn’t find relevant instructions in the document search. Does this `scrollbar` style require a plug-in?

![Snipaste_2021-12-14_02-37-16](https://user-images.githubusercontent.com/47672657/145869303-c750f28a-c887-4cc9-b747-957d811f4c01.png)

**Comment by lukewarlow** [comment]
There aren't scrollbar styles built into Tailwind at the minute. The docs are using custom written utilities which can be found [here](https://github.com/tailwindlabs/tailwindcss.com/blob/58dee72caa1a70d793baa386a3f2320935f3b8d1/src/css/utilities.css#:~:text=%40tailwind%20utilities%3B-,.scrollbar%2Dnone%20%7B,%7D,-.bg%2Dcheckered%20%7B).

The `supports-scrollbars` variant can be found in the tailwind config file in that repository too. It's worth noting those custom styles largely only apply to webkit based browsers (Chrome and Safari) NOT Firefox.
